joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/OperatorTests.scala

```diff
@@ -733,4 +710,15 @@ class OperatorTests extends PhpCode2CpgFixture {
       absCall.signature shouldBe s"${Defines.UnresolvedSignature}(1)"
     }
   }
+
+  "array/list unpacking should be lowered to several assignments" in {
+    val cpg = code("""<?php
+        |list(list($a, $b), $c) = $arr;
+        |""".stripMargin)
+
+    inside(cpg.call.nameExact(Operators.assignment).l) { case assignments: List[Call] =>
+      val targets = assignments.flatMap(_.argumentOption(1).code)
+      targets should contain allOf ("$a", "$b", "$c")
+    }
+  }
```

Comment:
This test shows that all of the relevant variables appear as call targets, but doesn't really test the structure of the AST being created. I would like to see a test like the placeholder which was removed which tests everything step-by-step. These are cumbersome to write, but serve a dual purpose as a test and documentation, so it's worth it in my opinion.